fix(mobile): removed iOS install content (#13836)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx

import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'

> The New Relic Cordova agent monitors native applications (iOS, Android) built on the [Apache Cordova framework](https://cordova.apache.org/docs/en/latest/guide/overview/index.html). Written in JavaScript, the Cordova agent also includes all New Relic mobile monitoring features for native apps. Once installed, the Cordova agent can:

Reviewer: ```suggestion
The New Relic Cordova agent monitors native applications (iOS, Android) built on the [Apache Cordova framework](https://cordova.apache.org/docs/en/latest/guide/overview/index.html). Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once installed, the Cordova agent can:
```

import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'

Our New Relic Cordova agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:

* **Capture JavaScript errors** Identify and fix problems quickly.
* **Track network requests:** See how your app interacts with the backend.
* **Use distributed tracing:** Drill down into handled exceptions and find the root cause.
* **Create custom events and metrics:** Understand how your users interact with your app.

## Compatibility requirements [#requirements]

> Before you install the Cordova agent, ensure your Flutter app meets these requirements:

Reviewer: ```suggestion
Before you install the Cordova agent, ensure your app meets these requirements:
```

* Capture interactions and their sequences
* Track user sessions

## Compatibility requirements [#requirements]

Before you install the Cordova agent, ensure your Flutter app meets these requirements:

* [Android-native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
* [iOS-native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)

## Install [#install]

> To install the Cordova agent:
> 
> * Follow our [guided install](https://onenr.io/08jqaJD9pjl), located directly in the UI.

Author: ```suggestion
To install the Cordova agent, follow our [guided install](https://onenr.io/08jqaJD9pjl), located directly in the UI.
```
